core/src/components/toast/toast.tsx

```diff
@@ -5,7 +5,7 @@ import { generateUniqueId, hasSlot } from '../../utils/utils';
 /**
  * @slot header - Slot for the Toast header.
  * @slot subheader - Slot for the Toast subheader.
- * @slot bottom - Slot for the Toast bottom section, used for links.
+ * @slot actions - Slot for the Toast bottom section, used for links.
```

Comment:
No link content in Toast component, looks like it still uses the "bottom" slot in the stories file?
![image](https://github.com/scania-digital-design-system/tegel/assets/48509966/63f0d068-a6e5-4dd6-84ff-494b2c6588f8)